config/data_scrubber.go

```diff
@@ -0,0 +1,62 @@
+package config
+
+import (
+	"regexp"
+	"strings"
+)
+
+var (
+	defaultSenstiveWords = []string{"password", "passwd", "mysql_pwd", "access_token", "auth_token", "api_key", "apikey", "secret", "credentials", "stripetoken"}
+)
+
+type DataScrubber struct {
+	Enabled               bool
+	SensitiveWords        []*regexp.Regexp
+	DefaultSensitiveWords []*regexp.Regexp
+	CustomSensitiveWords  []*regexp.Regexp
+}
+
+func NewDefaultDataScrubber() *DataScrubber {
+	newDataScrubber := &DataScrubber{
+		Enabled:               true,
+		SensitiveWords:        CompileStringsToRegex(defaultSenstiveWords),
+		DefaultSensitiveWords: CompileStringsToRegex(defaultSenstiveWords),
+		CustomSensitiveWords:  make([]*regexp.Regexp, 0, 0),
```

Comment:
Is there any advantage to having these all separated and duped (in case of the defaults)? Could we just have a single list of regexps that is extended with the custom ones?

Reply:
I used them separately to be able to enable/disable the default words but since we've changed the behavior of the DataScrubber, you're right that it doesn't make sense to maintain two separated lists. I'll merge them in a single one.